front/lib/iam/session.ts

```diff
@@ -83,39 +83,74 @@ export async function getUserFromSession(
   };
 }
 
+export type AuthLevel = "none" | "user" | "superuser";
```

Comment:
Maybe find another name that does not involve "Auth" as this is otherwise easy to confuse with Authenticator Roles? If no good other name, let's add a comment here to clarify that.

Reply:
Went with `UserPrivilege`.